src/sysadmin.cpp

```diff
@@ -103,6 +108,16 @@ int main(int argc, const char* argv[])
             protocol->Send(errResponse);
         });
 
+#ifdef _SYSADMIN_USE_SD_NOTIFY
+    reactor.add([]() {
```

Comment:
I know it's a little gross, but can we do this with `CallSoon` directly, instead of with `add`? `add` specifically is what refuses to compile on OSX